<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoUi</class>
 <widget class="QWidget" name="InfoUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="typeLA">
       <property name="text">
        <string>Infor&amp;mation Type:</string>
       </property>
       <property name="buddy">
        <cstring>typeCO</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="typeCO">
       <property name="toolTip">
        <string>Select the type of information to be output. Then specify the requested information below.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="infoLW"/>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="fixDateLA">
       <property name="text">
        <string>&amp;Fix Date:</string>
       </property>
       <property name="buddy">
        <cstring>fixDateLE</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="fixDateLE">
       <property name="toolTip">
        <string>Here you can enter a fix date (in ISO format: YYYY-MM-DD)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="nameLA">
       <property name="text">
        <string>&amp;Custom:</string>
       </property>
       <property name="buddy">
        <cstring>nameLE</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLE"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <includes>
  <include location="local">qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
